docs/specifications/tests/DNSQueryAndResponseDefaults.md

```diff
@@ -0,0 +1,162 @@
+|OpCode        | Require value to be "response"           | X     |                      |
+|QR flag       | Require flag to be set                   | X     |                      |
+|AA flag       | -                                        |       | Defined in test case |
+|TC flag       | Re-query over TCP if set                 |       |                      |
+|RD flag       | ignore                                   |       |                      |
+|RA flag       | ignore                                   |       |                      |
+|AD flag       | ignore                                   |       |                      |
+|CD flag       | ignore                                   |       |                      |
+|RCODE         | -                                        |       | Defined in test case |
+|Query name    | ignore                                   |       |                      |
+|Query type    | ignore                                   |       |                      |
+|Query class   | Require value to be same as in the query |       | Normally "IN"        |
+|EDNS          | ignore                                   |       |                      |
+
+* Check against query name and query type is, by default, done against the values
+  in the query section in the query, not in the response.
+  
+* When fetching records from the answer section, these are the default criteria:
+  * Only records matching the query name and the query type are fetched. Any
+    RRSIG records meeting the next criterium are also fetched.
+  * RRSIG records with the same query name and covering the query type are
+    fetched.
+  * CNAME records are ignored unless the query type is CNAME.
+
+* When the test case specification states that CNAME chains are to be followed in
+  the the answer section, these are the default criteria for the chain to be
+  fetched:
+  * The first CNAME in the chain must match the query name.
+  * The last record in the chain must either be a CNAME or a record matching the
+    query type.
+  * For each owner name of the CNAME records in the chain there must not be any
+    additional CNAME records in the answer section (only one CNAME record per
+    owner name).
+
+* Authority and additional sections are by default ignored.
+
+
+## Default handling of an *EDNS Response*
+
+An *EDNS response* is a response to an *EDNS Query*. An *EDNS Response* inherits
+the default handling from a *DNS Response* except for the response items
+specified below. Unless specified in the test case specification, the items in
+the response are handled using the default handling.
+
+|Response Item |Default handling             | Comment                                                        |
+|:-------------|:----------------------------|:----------------------- ---------------------------------------|
+|EDNS          | Take note if OPT is missing | Further actions to be specified in the test case specification |
```

Comment:
There's something wrong with the formatting here.

Reply:
Fixed.